src/modules/feature-modules/admin/services/users-validation-rules.service.ts

```diff
@@ -7,16 +7,12 @@ import { CoreService } from '@app/base';
 import { UrlModel } from '@app/base/models';
 
 
-type AdminOperationType = 'LOCK_USER' | 'UPDATE_USER_ROLE' | 'CHANGE_UNIT';
-type AdminOperationRuleType = 'AssessmentUserIsNotTheOnlyOne' | 'LastAccessorUserOnOrganisationUnit' | 'LastAccessorFromUnitProvidingSupport';
-
 export type AdminValidationResponseDTO = {
   validations: {
-    rule: AdminOperationRuleType,
+    rule: 'AssessmentUserIsNotTheOnlyOne' | 'LastQualifyingAccessorUserOnOrganisationUnit' | 'LastUserOnOrganisationUnit' | 'NoInnovationsSupportedOnlyByThisUser',
```

Comment:
Shouldnt we use an enum for this?

Reply:
We don't need the values of this, just the type, so I haven't created for now. Maybe in the future if needed.